tests/bootstrap.php

```diff
@@ -20,7 +20,11 @@
 use Cake\Datasource\ConnectionManager;
 use Cake\Log\Log;
 
-require_once 'vendor/autoload.php';
+if (is_file('/vendor/autoload.php')) {
```

Comment:
Shouldn't this be `./vendor/autoload.php`? Won't `/` be the root of the filesystem?